Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.py

```diff
@@ -580,7 +580,7 @@ def create_report(file_hash, reports, file_info, format_='xml', verbose=False):
 
         res_pdf = http_request(get_report_uri, 'POST', headers=DEFAULT_HEADERS, params=params)
 
-        file_name = 'wildfire_report_' + file_hash
+        file_name = 'wildfire_report_' + file_hash + '.pdf'
```

Comment:
Is it always pdf format or can it be other formats

Reply:
@yuvalbenshalom 
WF allows a pdf/xml report formats only
if xml, i handle it in the code